frappe/public/js/frappe/form/layout.js

```diff
@@ -250,6 +250,12 @@ frappe.ui.form.Layout = class Layout {
 			// collapse sections
 			this.refresh_section_collapse();
 		}
+
+		document.activeElement.focus();
```

Comment:
`document.activeElement` can also [have `null` value](https://developer.mozilla.org/en-US/docs/Web/API/Document/activeElement#value)

better to add a condition to check if `document.activeElement` exists

Reply:
Done